admin/react/assets/src/dashboard-app/pages/welcome/EmptyState.js

```diff
@@ -0,0 +1,23 @@
+import { __ } from "@wordpress/i18n";
+import React from "react";
+
+const EmptyState = () => {
+	return (
+		<div className="inline text-center">
+			<span className="text-center mb-6">
+				<svg className="my-0 mx-auto" width="65" height="64" viewBox="0 0 65 64" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Same here @imnavanath regarding svg reuse

Reply:
Added separate task in backlog for this - https://brainstormforce.atlassian.net/browse/CFP-38